### PR TITLE
Minor Kiwi Editor workflow improvements

### PR DESCRIPTION
**Pull request by mdeniz**
After selecting an image at the templates page, redirect to the kiwi editor, only if we are dealing with a kiwi image.

In tabs, keep "Kiwi" as first option in tabs, as "Image Editor", and put the other tabs in "Advanced".

Fixed breadcrumbs, now it shows the Package link

### CODE REVIEW

File: src/api/app/controllers/webui/package_controller.rb [hunk 1]
```diff
@@ -558,7 +558,14 @@ def branch
                                 user: User.current.login)
 
     flash[:notice] = 'Successfully branched package'
-    redirect_to(package_show_path(project: created_project_name, package: created_package_name))
+
+    branched_package_object = Package.find_by_project_and_name(created_project_name, created_package_name)
+
+    if request.env["HTTP_REFERER"] == image_templates_url && branched_package_object.kiwi_image?
+      redirect_to(import_kiwi_image_path(branched_package_object.id))
```
Comment (ChrisBr) [inline src/api/app/controllers/webui/package_controller.rb]: Hmm, maybe we can change the route to also accept project and package name?
https://github.com/openSUSE/open-build-service/blob/master/src/api/config/routes.rb#L195
(I'm fine with doing it in a dedicated PR)

File: src/api/app/views/webui/kiwi/_tabs.html.erb [hunk 2]
```diff
@@ -10,17 +10,10 @@
 
 <div class="box-header header-tabs" id="package_tabs">
   <ul>
-    <%= tab 'kiwi', 'Kiwi', controller: 'kiwi/images', action: 'show', selected: true %>
+    <%= tab 'kiwi', 'Image Editor', controller: 'kiwi/images', action: 'show', selected: true %>
     <% if @image.package.name == "patchinfo" %>
```
Comment (ChrisBr) [inline src/api/app/views/webui/kiwi/_tabs.html.erb]: Do we need the patchinfo tab at all?

File: src/api/app/controllers/webui/package_controller.rb (hunk 1 above)
Comment (mdeniz) [reply src/api/app/controllers/webui/package_controller.rb]: Better in another PR 👍

File: src/api/app/views/webui/kiwi/_tabs.html.erb (hunk 2 above)
Comment (mdeniz) [reply src/api/app/views/webui/kiwi/_tabs.html.erb]: I guess so... is a C&P from the package tabs...

Comment (ChrisBr) [reply src/api/app/views/webui/kiwi/_tabs.html.erb]: Then drop it (don't think it makes sense in this context) or move it at least to the Advanced section as well!

Comment (mdeniz) [reply src/api/app/views/webui/kiwi/_tabs.html.erb]: Moved to Advanced

**Comment by ChrisBr** [comment]
@mdeniz tests are failing

**Comment by mdeniz** [comment]
@ChrisBr already know that... I was working on that yesterday night. And I'm still on it 👍

**Pull request merged**